delft/rhea/network.nix

```diff
@@ -0,0 +1,34 @@
+{
+  networking.hostId = "9cd372da";
+
+  services.resolved = {
+    enable = true;
+    fallbackDns = [
+      "185.12.64.1"
```

Comment:
I really see no reason to have this list be different from the others btw - let's fix it along the way maybe? Before we forget :p